schemas/manifest.schema.json

```diff
@@ -0,0 +1,455 @@
+            "additionalProperties": false,
+            "properties": {
+                "yunohost": {
+                    "type": "string",
+                    "pattern": "^>= ?[0-9]*\\.[0-9]*\\.[0-9]*"
+                },
+                "architectures": {
+                    "oneOf": [
+                        {
+                            "type": "string",
+                            "const": "all"
+                        },
+                        {
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                                "enum": ["amd64", "i386", "armhf", "arm64"]
+                            }
+                        }
+                    ]
+                },
+                "multi_instance": {"$ref": "#/$defs/boolean_or_not_relevant"},
+                "ldap": {"$ref": "#/$defs/boolean_or_not_relevant"},
+                "sso": {"$ref": "#/$defs/boolean_or_not_relevant"},
+                "disk": {"$ref": "#/$defs/byte_size"},
+                "ram": {
+                    "type": "object",
+                    "required": [],
+                    "additionalProperties": false,
+                    "properties": {
+                        "build": {"$ref": "#/$defs/byte_size"},
+                        "runtime": {"$ref": "#/$defs/byte_size"}
+                    }
+                }
+            }
+        },
+        "install": {
+            "type": "object",
+            "required": [],
+            "additionalProperties": false,
+            "patternProperties": {
+                "^[a-z][a-z0-9_]*$": {
+                    "type": "object",
+                    "required": ["type"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "enum": [
+                                "string", "text", "select", "tags", "email", "url", "date", "time",
+                                "color", "password", "path", "boolean", "domain", "user", "group",
+                                "number", "range", "alert", "markdown", "file", "app"
+                            ]
+                        },
+                        "default": {
+                            "type": ["string", "number", "boolean"]
+                        },
+                        "ask": {"$ref": "#/$defs/translated_string"},
+                        "help": {"$ref": "#/$defs/translated_string"},
+                        "choices": {
+                            "type": "object",
+                            "required": [],
+                            "additionalProperties": false,
+                            "patternProperties": {
+                                "^[a-z][a-z0-9_]*$": {
+                                    "type": "string"
+                                }
+                            }
```

Comment:
Unfortunately we should also allow regular list of strings i think ? :grimacing:  Dunno how complex it is to allow both kind of structures

Reply:
It's not complex at all, actually I support both string and list of strings for the apt packages already :)

Reply:
Ah I already fixed it but not pushed yet ^^